memphis/consumer.py

```diff
@@ -53,7 +53,45 @@ def set_context(self, context):
         self.context = context
 
     def consume(self, callback):
-        """Consume events."""
+        """
+        Consume events.
```

Comment:
I think you can remove this line.  Your one-sentence description below is perfect.